image_width: 320
image_height: 240
camera_name: camera
camera_matrix:
  rows: 3
  cols: 3
  data: [403.97207,   0.     ,  93.66487,
           0.     , 437.70163,  -9.97263,
           0.     ,   0.     ,   1.     ]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [-0.163749, -0.011211, -0.040569, -0.000384, 0.000000]
rectification_matrix:
  rows: 3
  cols: 3
  data: [1., 0., 0.,
         0., 1., 0.,
         0., 0., 1.]
projection_matrix:
  rows: 3
  cols: 4
  data: [384.04553,   0.     ,  90.0279 ,   0.     ,
           0.     , 374.5799 , -14.96722,   0.     ,
           0.     ,   0.     ,   1.     ,   0.     ]
